src/content/en/updates/2019/01/webcomponents-v0-deprecations.md

```diff
@@ -0,0 +1,99 @@
+
+A part of web interoperability for [Web Components v1](/web/updates/2017/01/webcomponents-org),
+Chrome is removing
+[Shadow DOM v0](https://www.html5rocks.com/en/tutorials/webcomponents/shadowdom/),
+[Custom Elements v0](https://www.html5rocks.com/en/tutorials/webcomponents/customelements/)
+ and [HTML Imports](https://www.html5rocks.com/en/tutorials/webcomponents/imports/)
+  in Mar, 2019 at Chrome 73.
+
+[Intent to Deprecate](https://groups.google.com/a/chromium.org/forum/#!msg/blink-dev/h-JwMiPUnuU/sl79aLoLBQAJ)
+
+
+## Motivation
+
+Shadow DOM v0, Custom Elements v0, and HTML Imports (hereinafter, called v0 features)
+ were launched in 2014, but
+they did not get adopted by other browser engines. Instead, Shadow DOM v1,
+Custom Elements v1, and ES modules are widely adopted by various browser engines
+today. Chrome shipped Shadow DOM v1 / Custom Elements v1 in 2016 and ES
+modules in 2017.  
+Chrome is removing all of the v0 APIs at the same time because we've found the APIs are often used together. Removing them all at the same time will help developers migrate to the V1 APIs.
+
+- Shadow DOM v0: [Chromestatus Tracker](https://www.chromestatus.com/feature/4507242028072960) &#124;
+ [Chromium Bug](https://crbug.com/671907)
+- Custom Elements v0: [Chromestatus Tracker](https://www.chromestatus.com/feature/4642138092470272)  &#124;
+ [Chromium Bug](https://crbug.com/660759)
+- HTML Imports: [Chromestatus Tracker](https://www.chromestatus.com/feature/4507242028072960)  &#124;
+[Chromium Bug](https://crbug.com/766694)
+
+## Migration support
+If your site and/or library is using this features, you should upgrade them to
+the v1 APIs.
+Use `$ chrome --disable-blink-features=ShadowDOMV0,CustomElementsV0,HTMLImports`
+on command line to launch Chrome disabled v0 features for testing if your site works without those APIs.
+
+### Polyfill
+
+Now webcomponents polyfill supports v0 features on the all-v0-disabled Chrome. A web site using the polyfill should continue to work.
+This is also another option for the web authors.
+You can install the polyfill on the site using v0 features directly to keep
+it working.
+There are also other alternative polyfills for each feature (e.g.
+[AshleyScirra’s HTML Imports polyfill](https://github.com/AshleyScirra/html-imports-polyfill)).
+
+
+### Reverse Origin Trial
+
+For web authors who need more time to handle this removal, you can re-enable
+these features with an [Origin Trial](https://github.com/GoogleChrome/OriginTrials/blob/gh-pages/developer-guide.md).
+With a token generated for the trial, your site can continue to use them for
+an additional year, until Chrome 81.
+Chrome 81 is the planned milestone when we remove implementation code entirely so
+the trial token might last earlier.
+
+The trial name is `WebComponentsV0`. With that token, you can use Shadow DOM v0,
+Custom Elements v0, and HTML Imports APIs. Trial registration will be open
+after Chrome 73 beta release."
```

Comment:
I suggest simplifying the preceding two paragraphs to something like:

```suggestion
We plan to remove the features permanently in the Chrome 81 release, although the trial tokens may expire before then.

Soon after the Chrome 73 beta release, you can register for the trial at [Web Components V0](https://developers.chrome.com/origintrials/#/view_trial/2431943798780067841).
```
- The text above (with suggested edits) already explains that the trial will re-enable all 3 features, so probably don't need to repeat that here.
- I've added the link to the specific trial. This link will not work until the trial is made public, but it will redirect to the list of trials in the meantime.